bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/protocol/SetDimmerLevelCommand.java

```diff
@@ -52,6 +58,10 @@ public String getCommand() {
             args.add(String.valueOf(fadeSec));
         }
 
+        if (system == 1 || system == 2) {
```

Comment:
This block of code could maybe be pushed to RadioRACommand and reused throughout